packages/shared/utils/index.ts

```diff
@@ -119,3 +119,7 @@ export function objectEntries<T extends object>(obj: T) {
 export function getLifeCycleTarget(target?: any) {
   return target || getCurrentInstance()
 }
+
+export function deepClone<T>(obj: T): T {
+  return JSON.parse(JSON.stringify(obj))
```

Comment:
This is not a safe clone, and the result might not be as expected.

Please follow the pattern like this: 
https://github.com/vueuse/vueuse/blob/5f584701a2247509d009d7e39264c7dc14fcfc97/packages/core/useRefHistory/index.ts#L32-L39